manual/Makefile

```
@@ -40,3 +43,7 @@ clean:
 	$(MAKE) -C manual clean
 	$(MAKE) -C tools  clean
 	$(MAKE) -C tests  clean
+
+.PHONY: distclean
+distclean:
```

My bad I think - this should be:

```suggestion
distclean: clean
```